src/NovelRT.Interop/Audio/NrtAudioService.cpp

```diff
@@ -83,7 +129,16 @@ NrtResult Nrt_AudioService_playMusic(NrtAudioService service, NrtAudioServiceIte
     }
 
     auto serv = reinterpret_cast<Audio::AudioService*>(service);
-    serv->playMusic(reinterpret_cast<NovelRT::Audio::SoundBank::iterator&>(handle), loops);
+
+    try
+    {
+      serv->playMusic(reinterpret_cast<NovelRT::Audio::SoundBank::iterator&>(handle), loops);
+    }
+    catch (const Exceptions::NotInitialisedException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?